addons/point_of_sale/views/product_tag_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Product Tags -->
+    <record id="product_tag_form_view_inherit_point_of_sale" model="ir.ui.view">
+        <field name="name">product.tag.form.inherit.point.of.sale</field>
+        <field name="model">product.tag</field>
+        <field name="inherit_id" ref="product.product_tag_form_view"/>
+        <field name="priority" eval="100"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="description"/>
+            </field>
+        </field>
+    </record>
+
+    <menuitem id="pos_menu_products_tag_action" action="product.product_tag_action"
+              parent="point_of_sale.pos_menu_products_configuration" sequence="3"/>
```

Comment:
The menu item is not part of the specs. I can that it'll be useful, maybe only show during debug mode?

Reply:
(It's internally agreed with the PO.)